### DISCUSSION TITLE
ToggleButton

### DISCUSSION THREAD

**Discussion by coydetherage** [react-hook-form/react-hook-form#2271] [Q&A] [answered]
At first, react-hook-form is amazing. That made my work much easier!  Thanks you all, guys!

It is half working.  If you click in the box without touching label, it is working and it receives ToogleTest's value But if you click on label in the box, it didnt work and ToogleTest's value was undefined. I thought I can get it workaround by using selected={true} but I couldnt. Any suggestions?

https://codesandbox.io/s/strange-panini-jy8j1?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
https://codesandbox.io/s/green-flower-n8k8t?file=/src/App.js
